### DISCUSSION TITLE
What's the differences between open set recognition and out of distribution detection?

### DISCUSSION THREAD

**Discussion by xyy19920105** [iCGY96/awesome_OpenSetRecognition_list#18] [Q&A]
i'm a beignner of open set recognition. Could you tell me what is the differences between open set recognition and out of distribution detection?

**Comment by gary23ai** [comment]
OOD detection is similar to the rejection of unknown classes in OSR but does not require keeping the accuracy of known classes. However, more and more works evaluate both tasks together.

I hope it can help you!

**Reply by xyy19920105** [reply]
thanks for your reply
